pages/item/_.vue

```diff
@@ -135,7 +135,7 @@
                   </b-card-text>
                 </b-tab>
                 <b-tab
-                  v-if="showTranscription"
+                  v-if="mediaTranscription.length >= 1"
```

Comment:
Can we use the strict equality operator `>==`

Reply:
I am not sure if this exists in JavaScript.

Reply:
@mirjamverloop it exists in Javascript, it's works the same as >= but avoids type conversion. so for example 
1 == "1" // true
1 === "1" // false
https://stackoverflow.com/a/359509/2474466